Presentation.Web/app/components/it-system/it-system-catalog.controller.ts

```diff
@@ -705,6 +705,15 @@
                         attributes: { "class": "might-overflow" },
                         hidden: true
                     }
+                ],
+                excelOnlyColumns: [
+                    {
+                        persistId: "numberOfUsages",
+                        title: "Antal anvendere",
+                        width: 95,
+                        template: dataItem => dataItem.Usages.length.toString(),
```

Comment:
husk at tjekke for null så: `dataItem.Usages. ? dataItem.Usages.length.toString() : ""`;

Med mindre den afleverer et tom array hvis der ingen usages er. Så er det nok fint nok :-)

Reply:
Den afleverer tomt array. Har testet med helt nyt system som jeg ikke havde rørt ved. Og der skrev den fint "0" i excel